Lib/test/test_dataclasses/__init__.py

```diff
@@ -1524,6 +1524,24 @@ class A(types.GenericAlias):
         self.assertTrue(is_dataclass(type(a)))
         self.assertTrue(is_dataclass(a))
 
+    def test_is_dataclass_for_subclass(self):
+        # Test to ensure that inheritance from a dataclass works correctly
+        # and that the subclass is also recognized as a dataclass.
+        @dataclass
+        class X:
+            y: int
+
+        class Z(X):
+            z: int = 0  # Adding a default field to demonstrate inheritance and dataclass behavior
```

Comment:
We should just remove this attribute; it's not related to testing `is_dataclass` (and the current description of it as a "field" is not correct)

Reply:
Fixed this !